Makefile

```diff
@@ -3,6 +3,9 @@ PWD = $(shell pwd)
 BUILD_FOLDER = $(PWD)/build/$(NAMESPACE)
 ROOT_DIR := $(or ${ROOT_DIR},$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST)))))
 CONTAINER_COMMAND := $(or ${CONTAINER_COMMAND},docker)
+ifeq ($(CONTAINER_COMMAND), docker)
+	CONTAINER_COMMAND = $(shell docker -v | cut -f1 -d' ')
```

Comment:
hmm, doesn't work for me
```
make: Docker: Command not found
```

Reply:
also, why do we need it?

Reply:
If you have `docker` pointing to podman, it otherwise thinks you are using docker, not podman. So this is a way to detect when you have podman posing as docker.